src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -8,7 +8,7 @@ interface BreadcrumbsProps {
   separator: string;
 }
 
-export const Breadcrumbs = ({ separator }: BreadcrumbsProps) => {
+export default function Breadcrumbs({ separator }: BreadcrumbsProps) {
```

Comment:
I am indifferent to which pattern we follow for named vs default exports. In this specific case with the Breadcrumbs component, I was just trying to follow the "if you're only exporting one thing, make it default" pattern that we have in most of the code.

Similarly, I made everything export a function rather than declaring a const and exporting it later. in code I don't really have a strong opinion here either, just picked a direction and went with it for consistency. I think exporting the function is _slightly_ cleaner in that it's slightly fewer characters. But I can see the benefit to having one export statement at the end of a file too.